src/index.ts

```diff
@@ -1,6 +1,10 @@
 import * as context from './context'
 
-export { setupWorker, composeMocks } from './setupWorker/setupWorker'
+export {
+  SetupWorkerApi,
```

Comment:
I think this has no purpose in exposing it publicly. Instead, move it to `glossary` and reference from there internally, or as a part of tests.